### PR TITLE
[PE-1054][PE-1056] Add support for tagging resources, filter, and sweep

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

**Comment by kevinkirkup** [comment]
@Try75 @jmaness @joshpoth-pureport This should be ready to review and merge now.

File: test-infr/prod/pureport-infra/connections/variables.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+variable "connections_network_href" {
+  description = "The HREF for the network we should be deploy the connections"
```
Comment (Try75) [inline test-infr/prod/pureport-infra/connections/variables.tf]: I think all of these descriptions could use some grammar help, plus I think there might be copy-n-paste problems as well. Please revisit all of these descriptions.

**Comment by Try75** [comment]
I skipped everything in vendor/*.

File: test-infr/prod/pureport-infra/connections/variables.tf (hunk 1 above)
Comment (kevinkirkup) [reply test-infr/prod/pureport-infra/connections/variables.tf]: Updated.

**Pull request merged**